### ISSUE TITLE
[v2] Pseudo classes with newlines break

### ISSUE THREAD

**Issue by richardkall** [styled-components/styled-components#466]
## Version
2.0.0-1

## Reproduction
http://www.webpackbin.com/EJggwCZuG

## Steps to reproduce
When using `injectGlobal`, try separating pseudo classes with a comma and newline:

```
a {
  color: blue;

  &:active,
  &:hover {
    color: yellow;
  }
}
```

## Expected Behavior
Should be parsed as `a:active,a:hover {...}`

## Actual Behavior
Is parsed as `a:active,:hover a {...}`

**Comment by mxstbr** [comment]
Awesome, thanks! @thysultan, over to you 😉

**Comment by wmertens** [comment]
This is due to minification probably? The newline is stripped before passing the CSS to stylis. In this case the newline should have been replaced with a space.

See also https://github.com/styled-components/babel-plugin-styled-components/issues/29

**Comment by thysultan** [comment]
I'll fix this in stylis. Pushing a patch in a bit.

**Comment by thysultan** [comment]
patched in stylis v1.0.7

**Comment by mxstbr** [comment]
That was super fast, thanks! :tada: will publish a new v2 pre-release with the new version.
